src/newspack-ui/scss/elements/_typography.scss

```
@@ -28,57 +32,57 @@
 		font-weight: normal;
 	}
 
-	&__font--2xs {
+	.newspack-ui__font--2xs {
```

To avoid repetition, this can be expressed as:
```suggestion
	& &__font--2xs {
```

---

Ah, that's nicer! Doing a quick update before merging -- thanks @adekbadek! 🙂

---

Fixed up in [a42970d](https://github.com/Automattic/newspack-plugin/pull/3496/commits/a42970dc42a8ec2ce16f5ae0d3ca798cee022637)!